project/SparkBuild.scala

```diff
@@ -742,7 +742,10 @@ object Unidoc {
       "-windowtitle", "Spark " + version.value.replaceAll("-SNAPSHOT", "") + " JavaDoc",
       "-public",
       "-noqualifier", "java.lang",
-      "-tag", """note:a:Note\:"""
+      "-tag", """example:a:Example\:""",
+      "-tag", """note:a:Note\:""",
```

Comment:
This precedence respects scaladoc.

<img width="457" alt="2016-11-20 3 07 35" src="https://cloud.githubusercontent.com/assets/6477701/20457476/cf4d1832-aecf-11e6-9f6c-4aabe9384816.png">

After this PR, javadoc looks like

<img width="464" alt="2016-11-20 3 10 39" src="https://cloud.githubusercontent.com/assets/6477701/20457480/e4c0a184-aecf-11e6-8eaa-f4a387af45a8.png">